src/Akka.Persistence.Sql/Query/SqlReadJournal.cs

```diff
@@ -52,15 +52,14 @@ public class SqlReadJournal :
             ExtendedActorSystem system,
             Configuration.Config config)
         {
-            var writePluginId = config.GetString("write-plugin");
```

Comment:
This is not deleted, we're using the new `WritePluginId` property in `ReadJournalConfig` instead.